content/docs/contextual-editing/router.mdx

```diff
@@ -74,8 +74,6 @@ const default defineConfig({
 
 Now when a document is clicked in the CMS we will be re-directed to the page in the site with visual editing.
 
-<CloudinaryVideo src="https://res.cloudinary.com/forestry-demo/video/upload/q_100,h_584/e_accelerate:-20/v1655919318/tina-io/docs/RoutMapperVid" />
```

Comment:
**question**: Why was this video not transferred?

Reply:
I cannot get access to this file

Reply:
You can access the file through https://tina.io/docs/contextual-editing/router/

Reply:
as per our conversation on teams - this is resolved